scripts/install.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
+
+OCDEV_VERSION="v0.0.1"
+DOWNLOAD_URL="https://github.com/redhat-developer/ocdev/releases/download/$OCDEV_VERSION"
+BINARY_NAME="ocdev"
+# CANCELLATION_TIMEOUT specifies the number of seconds the script will wait
+# for the user to cancel the execution before continuing on.
+CANCELLATION_TIMEOUT=15
+INSTALLATION_PATH="/usr/local/bin/"
+PRIVILEGED_EXECUTION="sh -c"
+
+SUPPORTED_PLATFORMS="
+darwin-amd64
+linux-amd64
+linux-arm
+windows-amd64
+"
+
+echo_stderr ()
+{
+    echo "$@" >&2
+}
+
+command_exists() {
+	command -v "$@" > /dev/null 2>&1
+}
+
+check_platform() {
+    kernel="$(uname -s)"
+    if [ "$(uname -m)" = "x86_64" ]; then
+        arch="amd64"
+    fi
+
+    platform_type="${kernel,,}-$arch"
```

Comment:
this is a problem on macos as. mac has bash 3.2 but this works only on bash 4.0< 

if you need convert string to lover-case use some other method, like  `tr` (`echo "$a" | tr '[:upper:]' '[:lower:]'`)

Reply:
do we need to support installation on mac using this script at all?
I don't think this is mentioned in #70, nor did I see other packages like docker (ref: https://get.docker.com/). brew works fine there right?

Reply:
brew is the same as packages for linux, and we are making this script for it.

This script should be generic and work everywhere. Detect what system it is.
if it is debian based use deb packages, if it is rpm based use rpm packages, if it is other linux copy bin to /usr/local/bin, if it is macos use brew...

Reply:
Okay. I don't have a mac to test this on :(
How about you add macOS detection once this is merged with deb and rpm?

Reply:
on mac  "`uname -s` == Darwin"

for mac first check  if brew is installed (if there is a `brew` command) fail if not, and if there is then run two commands that are in the readme.  

I'll test it and debug/fix if there are problems.

Reply:
added